zeitschrift-fur-medienwissenschaft.csl

```diff
@@ -20,493 +14,297 @@
     <category field="humanities"/>
     <issn>1869-1722</issn>
     <summary>Style for zfm. Zeitschrift für Medienwissenschaft. Citations in notes; no bibliography.</summary>
-    <updated>2012-09-27T22:06:38+00:00</updated>
+    <updated>2014-09-13T22:01:31+00:00</updated>
     <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
   </info>
-  <!--
-================================================================================
-   TODO
-================================================================================
--->
-  <!--
-================================================================================
-   TERMS
-================================================================================
--->
+  <!-- HINWEISE
+       ========
+       - Bei Zeitschriftenartikeln (article-journal, article-magazine) muss
+         der Verlagsort (publisher-place) ins Zotero-Feld "Extra" eingetragen
+         werden, und zwar nach dem Muster "{:publisher-place: Drexel Hill,
+         Pennsylvania}".
+       - Bei Filmen (motion_picture) muss das Land ins Zotero-Feld "Extra"
+	 eingetragen werden, und zwar dem Muster "{:publisher-place: USA}".
+       - Das Originalpublikationsjahr muss ins Zotero-Feld "Extra" eingetragen
+         werden, und zwar nach dem Muster "{:original-date: 1964}".
+       - Bei Reihen (series) muss die Abkürzung "Bd. " als Präfix für die
+	 "Nummer der Reihe" ins Zotero-Feld "Nummer" eingetragen werden (nach
+	 dem Muster "Bd. 7").
```

Comment:
The style sheet calls for different abbreviations ("Bd.", "Vol." ...)
according to the language of the cited work. Can this (i.e.
multilingual citations) be done with CSL?

> On 2015-01-04T14:47:16+0100, Philipp Zumstein said:
> -   TERMS

> -================================================================================
> --->
> +  <!-- HINWEISE
> +       ========
> +       - Bei Zeitschriftenartikeln (article-journal, article-magazine) muss
> +         der Verlagsort (publisher-place) ins Zotero-Feld "Extra" eingetragen
> +         werden, und zwar nach dem Muster "{:publisher-place: Drexel Hill,
> +         Pennsylvania}".
> +       - Bei Filmen (motion_picture) muss das Land ins Zotero-Feld "Extra"
> +	 eingetragen werden, und zwar dem Muster "{:publisher-place: USA}".
> +       - Das Originalpublikationsjahr muss ins Zotero-Feld "Extra" eingetragen
> +         werden, und zwar nach dem Muster "{:original-date: 1964}".
> +       - Bei Reihen (series) muss die Abkürzung "Bd. " als Präfix für die
> +	 "Nummer der Reihe" ins Zotero-Feld "Nummer" eingetragen werden (nach
> +	 dem Muster "Bd. 7").

> I don't understand this part. The abbreviation "Bd." can be achived
> by `<text term="volume" form="short"/>` for de-DE. Are you
> suggesting to save "7" in one field and "Bd." in another field in
> Zotero?

> --- Reply to this email directly or view it on GitHub:
> https://github.com/citation-style-language/styles/pull/1313/files#r22439307

--
Till A. Heilmann, Dr. phil.

Online-Texte http://www.tillheilmann.info/german.php#OnlineTexts

Universität Siegen
Philosophische Fakultät
Medienwissenschaftliches Seminar
Adolf-Reichwein-Straße 2, Raum AR-K 611
57076 Siegen
Telefon: +49 271 740-4791
Telefax: +49 271 740-2731
www.uni-siegen.de/phil/medienwissenschaft/
www.multimediale-systeme.de